src/CMakeLists.txt

```
@@ -97,6 +97,13 @@ add_subdirectory(app)
 
 set(DATA_OUTPUT_DIR ${CMAKE_BINARY_DIR}/bin/data)
 
+if(ENABLE_ASEPRITE_EXE)
+  set(main_target aseprite)
+  if(APPLE)
+    set(DATA_OUTPUT_DIR ${CMAKE_BINARY_DIR}/bin/${main_target}.app/Contents/Resources/data)
```

There is an issue here where the output directory should be `Aseprite.app` (not `aseprite.app`): 
```suggestion
    set(DATA_OUTPUT_DIR ${CMAKE_BINARY_DIR}/bin/Aseprite.app/Contents/Resources/data)
```